### PR TITLE
Added SES.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: load_to_rds/create_html_message.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        body_in_tag += f"</{tag_name}>"
+    return body_in_tag
+
+
+def format_price(price: int) -> str:
+    """Formats the price correctly."""
+    if price == 0:
+        return "FREE"
+    price_str = round(price / 100, 2)
+    return "£" + str(price_str)
```
Comment (Leon-JS02) [inline load_to_rds/create_html_message.py:19]: This could be an f string here

File: load_to_rds/email_subscribers.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from create_html_message import create_html
+from get_subscriber_emails import get_subscribers_per_genre
+
+load_dotenv()
+
+
+def format_genre_text(genre: str) -> str:
+    """Makes genre representable in an email."""
+
+    if genre.lower() == "rpg":
```
Comment (Leon-JS02) [inline load_to_rds/email_subscribers.py:18]: Interesting edge case. This works, but is there maybe a more elegant solution?

File: load_to_rds/get_subscriber_emails.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+from os import environ as ENV
+
+from boto3 import client
+from dotenv import load_dotenv
+
+from create_topic_message import create_text_message
+
+load_dotenv()
+
+# pylint: disable = C0103
```
Comment (Leon-JS02) [inline load_to_rds/get_subscriber_emails.py:12]: What's C0103?

**Review by Leon-JS02 — DISMISSED** [review]
Decent code, glad it's working. Could always do with a refactor/clean up for future improvements.

File: load_to_rds/get_subscriber_emails.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    """Returns all of the subscriber emails for each genre."""
+
+    sns = client(service_name="sns",
+                 aws_access_key_id=ENV["AWS_ACCESS_KEY"],
+                 aws_secret_access_key=ENV["AWS_SECRET_ACCESS_KEY"])
+
+    topics = get_SNS_topics(sns)
+
+    subscribers_per_genre = []
+    for topic in topics:
```
Comment (MichaelHassan) [inline load_to_rds/get_subscriber_emails.py:47]: could this be in a function of itself.

File: load_to_rds/email_subscribers.py
Comment (MichaelHassan) [inline load_to_rds/email_subscribers.py:1]: You have forgotten to call it in the lamdba handler

**Review by paulauus — APPROVED** [review]
I guess you're also going to add the table at a later stage?

File: load_to_rds/email_subscribers.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    for website in scraped_data:
+        for game in website["listings"]:
+            if any(genre in game_genre.lower()
+                   for game_genre in game["genres"]):
+                genre_games.append(game)
+
+    return genre_games
+
+
+def send_genre_emails(scraped_data: list[dict]):
```
Comment (MichaelHassan) [inline load_to_rds/email_subscribers.py:38]: This is quite a big function can this be chopped up.

**Review by MichaelHassan — APPROVED** [review]
Looks good but, I would say look at my comments and resolve them.

**Review by Leon-JS02 — APPROVED** [review]
Re-approved after Mehrge conflict.

**Review by MichaelHassan — APPROVED** [review]
APPROVED

**Pull request merged**